app/src/panels/meter/panel-meter.vue

```diff
@@ -0,0 +1,195 @@
+<template>
+	<div class="panel-meter" :class="[{ 'has-header': showHeader }, `size-${size}`]">
+		<output>{{ displayValue }}</output>
+		<svg :width="svgSize.width" :height="svgSize.height">
+			<circle
+				cx="50%"
+				:cy="size === 'half' ? '100%' : '50%'"
+				fill="none"
+				:stroke-width="strokeWidth"
+				:stroke-linecap="roundedStroke ? 'round' : 'inherit'"
+				:r="radius"
+				stroke="var(--background-subdued)"
+			/>
+			<circle
+				cx="50%"
+				:cy="size === 'half' ? '100%' : '50%'"
+				fill="none"
+				:stroke-width="strokeWidth"
+				:stroke-linecap="roundedStroke ? 'round' : 'inherit'"
+				:r="radius"
+				:stroke="conditionalColor"
+			/>
+		</svg>
+	</div>
+</template>
+
+<script lang="ts" setup>
+import { computed, unref } from 'vue';
+import { useI18n } from 'vue-i18n';
+
+interface Props {
+	showHeader: boolean;
+	collection: string;
+	field: string;
+	width: number;
+	height: number;
+	fn: 'avg' | 'avgDistinct' | 'sum' | 'sumDistinct' | 'count' | 'countDistinct' | 'min' | 'max';
+	data?: { [fn: string]: { [field: string]: number } }[];
+	size?: 'full' | 'half';
+	decimals?: number;
+	strokeWidth?: number;
+	roundedStroke?: boolean;
+	color?: string;
+	max?: number;
+	conditionalFill?: { operator: '=' | '!=' | '>' | '>=' | '<' | '<='; color: string; value: number }[];
+}
+
+const props = withDefaults(defineProps<Props>(), {
+	decimals: 0,
+	data: () => [],
+	strokeWidth: 20,
+	roundedStroke: false,
+	color: 'var(--primary)',
+	max: 100,
+	conditionalFill: () => [],
+	size: 'full',
+});
+
+const { n } = useI18n();
+
+const percent = computed(() => {
+	if (props.data.length === 0) return 0;
+	if (!props.fn || !props.field) return 0;
+	return +props.data[0][props.fn][props.field] / +props.max;
+});
+
+const displayValue = computed(() => {
+	const percentDecimal = Number((unref(percent) * 100).toFixed(1));
+	return n(percentDecimal) + '%';
+});
+
+const radius = computed(() => {
+	const widthPx = props.width * 20 - 40 - 4;
+	let heightPx = props.height * 20 - 40 - 4;
```

Comment:
Was able to track down there was originally comment that describe what they are, but was unintentionally removed in later iterations. I've re-added the comment for clarity, and opting to keep `- 40 - 4` instead of `- 44` with the added comment in mind 👍